--- a/application/basilisk/confvars.sh	2017-03-09 09:11:36.643104656 +0100
+++ b/application/basilisk/confvars.sh	2017-03-09 11:13:13.798231732 +0100
@@ -14,6 +14,16 @@
   MOZ_BUNDLED_FONTS=1
 fi
 
+
+# Reduce data sharing with Mozilla
+MOZ_DATA_REPORTING=0
+MOZ_TELEMETRY_REPORTING=0
+MOZ_CRASHREPORTER=0
+MOZ_SERVICES_HEALTHREPORT=0
+
+
+
+
 if test "$OS_ARCH" = "WINNT"; then
   MOZ_MAINTENANCE_SERVICE=
 fi
